src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.js

```diff
@@ -62,35 +37,13 @@ class BaseSidebarScreen extends Component {
         Welcome.show({routes, showCreateMenu: this.showCreateMenu});
```

Comment:
But I'm sure [this](https://github.com/Expensify/App/pull/12369#issuecomment-1310452390) will fail for all platforms.

As this code is in the wrong place, `this.showCreateMenu` is not defined here!

Reply:
Oh that is a great catch.

@hannojg can you update this today?

Reply:
@Santhosh-Sellavel So this is the last thing which is holding us from merging right?

Reply:
Yeah once the changes are done, I'll test again if no issues we are good to Merge!

Reply:
Yeah, good catch thx! Got it fixed now, when singing up for the first time the FAB actions are opening!